azure-pipelines.yml

```diff
@@ -158,12 +158,12 @@ jobs:
   condition: and(ne(variables['Build.Reason'], 'PullRequest'), eq(variables['Build.SourceBranch'], 'refs/heads/master'))
```

Comment:
Shouldn't the condition depend on `mtrx.projplatforms`?

Reply:
Ahhh.... hmmm, yes, I think you're right.  We don't want to try and register if nothing was changed.